azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/WebAppUtils.java

```diff
@@ -47,18 +47,22 @@ public static WithNewAppServicePlan defineApp(final AbstractWebAppMojo mojo) thr
                 withGroup.withNewResourceGroup(resourceGroup);
     }
 
-    public static DockerImageType getDockerImageType(final ContainerSetting containerSetting) {
+    public static ImageType getImageType(final ContainerSetting containerSetting) {
```

Comment:
question, since the method has been renamed to be a broader concept than dockerimage, why the parameter is containerSetting?

Reply:
Emm.., actually they are still docker images. But user will not aware of that from the portal UI. So I change the name here. 

According to Rome's comments, I have changed back to the origin name.